### PR TITLE
adding getting started guide content

### PR DESCRIPTION
**Pull request by stephamon**
Created folders for common getting started guide content and Auto Scale API examples. Builds with no errors and checks out with the deconst client.

### CODE REVIEW

**Comment by stretchservers** [comment]
Can one of the admins verify this patch?

**Comment by stretchservers** [comment]
Can one of the admins verify this patch?

**Comment by manishtomar** [comment]
@stephamon Kindly fix lint errors :) https://travis-ci.org/rackerlabs/otter/builds/95421421#L390-L391

**Comment by meker12** [comment]
@manishtomar  I merged this without checking.  Will fix lint errors and check in ASAP

**Comment by manishtomar** [comment]
@meker12 It will be nice to add deconst setup to CI/CD (travis). This would require adding `deconst` to requirements and building it everytime by having an entry in Makefile and invoking it. Would it be possible for you to do this? I can help if required.

**Comment by meker12** [comment]
@manishtomar   So @smashwilson  set up a Strider build that runs deconst and deploys the new content as soon as it's available -- pretty fast.  It's here:  http://ci.deconst.horse/rackerlabs/otter/

I'd be interested to know what it would take to include Nexus in your repo.  I think @smashwilson has answers.

**Comment by smashwilson** [comment]
> I'd be interested to know what it would take to include Nexus in your travis/build set up.

Well, the Deconst Sphinx preparer uses Python 3, so it wouldn't be trivial. We'd need to backport the preparer to 2.7 before we could realistically do that.

However: on my short list of changes for Deconst is to move all of the preparer builds _off_ of Travis (deconst/deconst-docs#1), partially for this exact reason: using Travis, each repository can have exactly one build. Environments and dependencies will inevitably collide somewhere. We'd already hit this with rack, which needs a Go environment.

We also can't securely support pull request builds (deconst/deconst-docs#7) on Travis without leaking credentials, because Travis stores the build configuration entirely in-repository and encrypted secrets are only available to builds triggered from the head fork.

In short, we could but we won't :wink: In the long term we'll likely have a separate GitHub status check  and a separate build badge for the health of the Deconst build.
